### ISSUE TITLE
make a CI test for alignments

### ISSUE THREAD

**Issue by kohlhase** [UniFormal/MMT#232]
there should be a test script in CI that runs after each commit and tries to load the alignments and complains if it fails (see https://gl.mathhub.info/alignments/Public/commit/1c525f79c359df72ecbe95d132969beee5d722be). 
Such a script should run for all student-fed repositories like that one.

**Comment by tkw1536** [comment]
I would like to have a closer look at the testing infrastructure in general before doing this.

**Comment by kohlhase** [comment]
yes, I think we should. I am seeing that @Jazzpirate  is held back by having to fix commits from other people without knowing what the changes were. This may not be urgent, but should be attacked at a bigger scale.

**Comment by tkw1536** [comment]
This causes a weird timeout on travis, probably to do with the size of NASA. Postponing to next release.

**Comment by tkw1536** [comment]
This will happen for the ODK / MiTM alignments during this week
